source/features/git-checkout-pr.tsx

```diff
@@ -105,7 +105,7 @@ void features.add(__filebasename, {
 	],
 	exclude: [
 		isMergedPR,
-		() => select.exists('#partial-discussion-header [title="Status: Draft"]')
+		isClosedPR
```

Comment:
…

this is different

Reply:
Look at the full diff. I made a mistake in one of the  commits and changed it to draft